tmt/scripts/create-results.sh

```
@@ -0,0 +1,75 @@
+# /**********************************************************************
```

Shellcheck (0.10.0)

[error] This shebang was unrecognized. ShellCheck only supports sh/bash/dash/ksh/'busybox sh'. Add a 'shell' directive to specify.

(SC1008)